packages/core/src/decorators/lazy-getter.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import { ApplicationError } from 'n8n-workflow';
+
+export function Lazy(
```

Comment:
Can we please include documentation what this decorator does

Reply:
done